src/plugins/image/image.yml

```diff
@@ -9,7 +9,7 @@ hoverOption:
   de: Hover
 revealOption:
   en: Reveal
-  de: Auflösung
+  de: Aufdeckung
```

Comment:
Maybe let's go with "Aufdecken" (similar to "Zuschneiden")

Reply:
Also aligns with "Hover" then, changed 👍

Reply:
Poifect!